tests/conftest.py

```diff
@@ -0,0 +1,12 @@
+import tempfile
+
+import pytest
+
+from ballet.compat import pathlib
+
+
+@pytest.fixture
+def tempdir():
```

Comment:
Mainly needs to be in tests/conftest.py for pytest to be able to discover it as a fixture.

It could be helpful in general usage, but in that case I might recommend just using tempfile.TemporaryDirectory context manager directly.